src/utils/loggable.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import os
+
+
+DEFAULT_LOG_LEVEL = logging.WARNING
+
+_COLOR_TMPL = "\033[3{}m"
+(RED,
+ GREEN,
+ YELLOW,
+ BLUE,
+ VIOLET,
+ CYAN) = map(lambda x: _COLOR_TMPL.format(x), range(1, 7))
+
+BOLD = "\033[1m"
+ESC = "\033[0m"
+
+
+def apply_style(text, color=None, bold=False):
+    bold = BOLD if bold else ""
+    color = color or ""
+    return "{}{}{}{}".format(bold, color, text, ESC)
+
+
+def get_log_level():
+    level = os.environ.get('LOGLEVEL')
```

Comment:
I would prefer a name more specific to hack-sound-server, e.g. `HACK_SOUND_SERVER_LOGLEVEL`.